### ISSUE TITLE
Regression in launching IntegrationTests instrumented with Jacoco-Maven-Plugin

### ISSUE THREAD

**Issue by bvarner** [quarkusio/quarkus#19623]
### Describe the bug

Quarkus 2.1.x (I'm not sure which version initiated it)

Trying to run integration tests (failsafe) with an ${argLine} provided from jacoco-maven-plugin that contains multiple arguments is **broken**. Looks like it's parsing the -javaagent argument for `,`'s then splitting those into additional java command arguments, which is, clearly not valid.

### Expected behavior

Expected behavior is for the ${argline} to not get mangled into separate arguments, and be quoted wholly as a single command line argument. Just like it was in 2.0.x.... below:

```
Executing [java, -javaagent:/home/bvarner/.m2/repository/org/jacoco/org.jacoco.agent/0.8.7/org.jacoco.agent-0.8.7-runtime.jar=destfile=/home/bvarner/Documents/work/AVT/spcd-rest-api/target/jacoco-quarkus.exec,append=true,exclclassloader=*QuarkusClassLoader, -Dquarkus.http.port=8081, -Dquarkus.http.ssl-port=8444, -Dtest.url=http://localhost:8081, -Dquarkus.log.file.path=/home/bvarner/Documents/work/AVT/spcd-rest-api/target/quarkus.log, -Dquarkus.log.file.enable=true, -Dquarkus.profile=it, -jar, /home/bvarner/Documents/work/AVT/spcd-rest-api/target/quarkus-app/quarkus-run.jar]
__  ____  __  _____   ___  __ ____  ______ 
 --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
 -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
--\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
```

Note, the `,append=true,exclclassloader=...` bits are arguments to the javaagent, _not_ the java executable.

### Actual behavior

Running integration tests with 2.1.x:
```
Executing [java, -javaagent:/home/bvarner/.m2/repository/org/jacoco/org.jacoco.agent/0.8.7/org.jacoco.agent-0.8.7-runtime.jar=destfile=/home/bvarner/Documents/work/AVT/spcd-rest-api/target/jacoco-quarkus.exec, append=true, exclclassloader=*QuarkusClassLoader, -Dquarkus.http.port=8081, -Dquarkus.http.ssl-port=8444, -Dtest.url=http://localhost:8081, -Dquarkus.log.file.path=/home/bvarner/Documents/work/AVT/spcd-rest-api/target/quarkus.log, -Dquarkus.log.file.enable=true, -Dquarkus.profile=it, -jar, /home/bvarner/Documents/work/AVT/spcd-rest-api/target/quarkus-app/quarkus-run.jar]
Error: Could not find or load main class append=true
```

### How to Reproduce?

https://quarkus.io/guides/tests-with-coverage#coverage-for-integration-tests

### Output of `uname -a` or `ver`

Linux fibonacci 5.11.0-31-generic #33-Ubuntu SMP Wed Aug 11 13:19:04 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

### Output of `java -version`

Linux fibonacci 5.11.0-31-generic #33-Ubuntu SMP Wed Aug 11 13:19:04 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

### Quarkus version or git rev

2.1.x

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f) Maven home: /home/bvarner/.m2/wrapper/dists/apache-maven-3.6.3-bin/1iopthnavndlasol9gbrbg6bf2/apache-maven-3.6.3 Java version: 11.0.11, vendor: Ubuntu, runtime: /usr/lib/jvm/java-11-openjdk-amd64 Default locale: en_US, platform encoding: UTF-8 OS name: "linux", version: "5.11.0-31-generic", arch: "amd64", family: "unix"

**Comment by geoand** [comment]
https://github.com/quarkusio/quarkus/pull/19641 should fix the issue but it's probably too late to get it into `2.2.0.Final`, which means it will likely be available from `2.2.1.Final`

**Comment by bvarner** [comment]
Yes, it's fixed in 2.2.1.Final.
